### PR TITLE
working on collapsable table rows

### PR DESCRIPTION
**Pull request by williamdingwall**

### CODE REVIEW

File: src/assets/toolkit/scripts/toolkit.js [hunk 1]
```diff
@@ -1,31 +1,70 @@
+//------------------------------------------------------------//
+
+// * Panels
```
Comment (saralohr) [inline src/assets/toolkit/scripts/toolkit.js]: Think this might be an interloper.

File: src/assets/toolkit/scripts/toolkit.js [hunk 2]
```diff
@@ -1,31 +1,70 @@
...
+  }
+
+}
+
+// * 4. Collapsable Panel Row
+
+u('.js-expandableControlDemo').on('click', function(e) {
+  e.preventDefault();
+  var childRows = u(this).data('control');
+  console.log(childRows)
```
Comment (saralohr) [inline src/assets/toolkit/scripts/toolkit.js]: Did you want to keep this in?

File: src/patterns/components/panel/collapsable-rows.hbs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+notes: |
+
+---
+
+
+
+<div class="Panel Panel--collapsable">
+    <div class="Panel-body">
+      <div class="Panel-row Panel-row--withCells">
+        {{#each (data "invoicing.column_headers")}}
```
Comment (saralohr) [inline src/patterns/components/panel/collapsable-rows.hbs]: Yay! Glad to see data finally being put to use!

File: src/assets/toolkit/scripts/toolkit.js (hunk 1 above)
Comment (williamdingwall) [reply src/assets/toolkit/scripts/toolkit.js]: yep, nice catch

File: src/assets/toolkit/scripts/toolkit.js (hunk 2 above)
Comment (williamdingwall) [reply src/assets/toolkit/scripts/toolkit.js]: na, it's gone

File: src/patterns/components/panel/collapsable-rows.hbs (hunk 3 above)
Comment (williamdingwall) [reply src/patterns/components/panel/collapsable-rows.hbs]: yeah, I liked playing with it. Wanted to get an idea of how it'd work coming from a data set - all those helpers are awesome too!

**Comment by williamdingwall** [comment]
Thanks! got all those updates in, both here and on teamsnap-ui

**Pull request merged**